dbt/adapters/athena/utils.py

```diff
@@ -11,3 +11,9 @@ def clean_sql_comment(comment: str) -> str:
 def get_catalog_id(catalog: Optional[DataCatalogTypeDef]) -> Optional[str]:
     if catalog:
         return catalog["Parameters"]["catalog-id"]
+
+
+def get_chunks(lst, n):
+    """Yield successive n-sized chunks from lst."""
+    for i in range(0, len(lst), n):
+        yield lst[i : i + n]
```

Comment:
let's add type annotations here

```suggestion
from typing import Iterable, List, TypeVar


T = TypeVar('T')

def get_chunks(lst: Iterable[T], n: int = 1) -> Iterable[List[T]]:
    """Yield successive n-sized chunks from lst."""
    for i in range(0, len(lst), n):
        yield lst[i : i + n]
```

Reply:
also let's add unit tests for this method

Reply:
`n=1` doesn't seem very natural default to me, so would omit that, otherwise agree and thanks for the typing 👍

Reply:
would use `Collection` instead of `Iterable` since an `Iterable` doesn't seem to be `Sized` 🤯 , c.f., https://stackoverflow.com/questions/49427944/typehints-for-sized-iterable-in-python